### ISSUE TITLE
Errors when libwrapper enabled

### ISSUE THREAD

**Issue by Zamrod** [caewok/fvtt-elevated-vision#2]
If libwrapper v1.12.8.0 is enabled then when you select a token, it throws the below error in V10 build 284. It then doesn't render the sight of the token at all.

```Uncaught TypeError: Cannot read properties of undefined (reading 'los')
[Detected 2 packages: elevatedvision, lib-wrapper]
    at DetectionModeBasicSight._testLOS (foundry.js:40326:23)
    at 🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:511:22)
    at DetectionModeBasicSight._testLOSDetectionMode (tokens.js:122:17)
    at 🎁DetectionMode.prototype._testLOS#0 (libWrapper-wrapper.js:187:52)
    at DetectionModeBasicSight._testPoint (foundry.js:40364:16)
    at foundry.js:40273:36
    at Array.some (<anonymous>)
    at DetectionModeBasicSight.testVisibility (foundry.js:40273:18)
    at 🎁call_wrapped [as call_wrapped] (libWrapper-wrapper.js:511:22)
    at DetectionModeBasicSight.testVisibilityDetectionMode (tokens.js:99:10)
    at 🎁DetectionMode.prototype.testVisibility#0 (libWrapper-wrapper.js:187:52)
    at CanvasVisibility.testVisibility (foundry.js:33122:39)
    at get isVisible [as isVisible] (foundry.js:45069:38)
    at CanvasVisibility.restrictVisibility (foundry.js:33045:68)
    at CanvasVisibility.refreshCanvasVisibilityPolygons (vision.js:89:8)
    at 🎁CanvasVisibility.prototype.refresh#0 (libWrapper-wrapper.js:187:52)
    at #update (foundry.js:41032:58)
    at t.emit (ticker.min.mjs:8:622)
    at e.update (ticker.min.mjs:8:3473)
    at _tick (ticker.min.mjs:8:1482)```

**Comment by caewok** [comment]
That's new! Thanks; I will look into it.

**Comment by caewok** [comment]
Fixed in v0.0.2.
